docs/testplan/dual_tor/dual_tor_active_active_test_plan.md

```diff
@@ -85,3 +85,26 @@ For downstream traffic from T1, the ECMP routing behavior is out of the scope of
+| normal operation scenario | Verify upstream traffic with SoC IP as src IP. | <br>1. Verify active-active setup. <br/><br>2. Start upstream traffic.<br/> | None |  Both ToR remain `active` and `healthy` in the whole test. | T1 receives all packets with no disruption. |
+| ToR-A link down | Verify SoC -> T1 traffic in ToR-A downstream interface down scenario | <br>1. Take down ToR to Server links on ToR-A<br/><br>2. Start upstream traffic. <br/> | Take down links between ToR-A and server NiC device | ToR-A toggles to `standby` after link down and toggles back to `active` after link up. | T1 receives packets with one disruption. |
+
+#### downstream traffic verification
+
+| <div style="width:200px">Case</div>           | <div style="width:300px">Goal</div>                                              | <div style="width:300px">Test Steps</div>                                                                          | <div style="width:300px">Action during I/O</div> | <div style="width:300px">Expected Control Plane</div>                                                      | <div style="width:300px">Expected Data Plane</div>                                                                                                                                                                       |
+| --------------------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------ | ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| normal operation scenario | Verify T1 -> ToR-A -> SoC traffic in normal operation scenario. | <br>1. Verify an active-active dualtor setup <br/><br>2. Start downstream I/O to ToR-A<br/> | None | Both ToR remain `active` and `healthy` in the whole test. |Server receives all packets with no disruption. |
+| normal operation scenario | Verify T1 -> ToR-B -> SoC traffic in normal operation scenario. | <br>1. Verify an active-active dualtor setup <br/><br>2. Start downstream I/O to ToR-B<br/> | None | Both ToR remain `active` and `healthy` in the whole test. |Server receives all packets with no disruption. |
+| ToR-A link down | Verify T1 -> ToR-A -> SoC traffic in ToR-A downstream interface down scenario | <br>1. Take down ToR to Server links on ToR-A<br/><br>2. Start downstream traffic to ToR-A<br/> | Take down links between ToR-A and server NiC device | ToR-A toggles to `standby` after link down and toggles back to `active` after link up. | Packets are forwarded back to T1 through IPinIP tunnel. Server receives packets with one disruption. |
```

Comment:
We should also add the case where ToR A is configured to be standby?

Reply:
Or we can have one case to verify that when ToR A is standby, tunnel route created, IO is forwarded. And other test cases to verify that all expected failure combos would lead to ToR A stand + tunnel route created.

Reply:
We have link down case for both upstream and downstream. I actually prefer link down case over configuring to standby, as in link down case if tunnel route wasn't added properly, traffic would drop completely. 

Is there any reason we want to add a configuration one?

Reply:
Configuration change is an important part of operations.

Reply:
> Configuration change is an important part of operations.

Updated accordingly. Test gap created: 
https://github.com/sonic-net/sonic-mgmt/issues/7735
https://github.com/sonic-net/sonic-mgmt/issues/7736

Tunnel-route creation and IO verification is covered in dualtor/orchagent tests.